package.json

```diff
@@ -8,6 +8,7 @@
   "scripts": {
     "build": "gulp build",
     "clean": "gulp clean",
+    "reinstall": "rm -rf node_modules && npm  --force cache clean && npm install",
```

Comment:
You can already use `npm run clean -- --modules --cache && npm install`
New npm command works against the last script purge to keep it lean and efficient.

Using a makefile to bundle the script primitives into a full process to do something can be the next step.
Adding more and more scriptlet to npm-scripts isn't the best way to achieve that as it gets messy over time.

Reply:
@debloper `npm run clean` uses `gulp` to clean/remove files. What if the user doesn't have globally installed `gulp` ?